### ISSUE TITLE
cpu does not pick fox

### ISSUE THREAD

**Issue by geranim0** [altf4/cpu-level-11#28]
followed your instructions, does not pick fox. selected the pipe in the controller setup.

**Comment by altf4** [comment]
What OS are you on exactly? I think that the location to MemoryWatcher and the named pipe controller input might be off for you.

The CPU is going to try to read a named pipe at ~./dolphin-emu/Pipes/. But that might be in a different location for you. (Maybe ~./local/share/dolphin-emu) 

This ought to be detected automatically, honestly. So I'll make this into a defect issue to keep track of that for sure.

**Comment by geranim0** [comment]
I'm on debian and I tried to modify your source to get the right locations
but it was unsuccessful. Also, the pipe controller configuration is
unclear, do we have to let it blank or import some kind of pipe.ini?

2016-01-15 2:13 GMT-05:00 AltF4 <notifications@github.com>:

**Comment by altf4** [comment]
Yea the instructions for fully setting up a controller are not obvious and more of a Dolphin config, so I thought it would be out of scope for my readme. But here's what my config looks like (you could also do this through the Dolphin controller setup GUI)

```
[GCPad2]
Device = Pipe/0/cpu-level-11
Buttons/A = `Button A`
Buttons/B = `Button B`
Buttons/X = `Button X`
Buttons/Y = `Button Y`
Buttons/Z = `Button Z`
Buttons/Start = `Button START`
Main Stick/Up = `Axis MAIN Y +`
Main Stick/Down = `Axis MAIN Y -`
Main Stick/Left = `Axis MAIN X -`
Main Stick/Right = `Axis MAIN X +`
C-Stick/Up = `Axis C Y +`
C-Stick/Down = `Axis C Y +`
C-Stick/Left = `Axis C X -`
C-Stick/Right = `Axis C X +`
Triggers/L = `Button L`
Triggers/R = `Button R`
```

Inside GCPadNew.ini

**Comment by rukai** [comment]
I thought the readme was saying that the p1 & p2 needed to be set to Wii U GC adapater.
When I read it again It's easy to see what you really meant, but if you specify p1 that would clear up any confustion.

Also, the GCPad2 configuration is super important and should be in the readme imo, no way to get it to work without it.

Other then that, was quite straight forward to setup.
Sweet project!

**Comment by altf4** [comment]
There is a mistake with the above controller INI file. It is missing the L and R analog settings. Fox will fail to light shield without it:

**Comment by altf4** [comment]
I just added a lot of logic in afc025555e28aa0c66be0c7c9f2a5aa39a63c92e that should fix this. Give it a try. All the setup around moving files and stuff is done automatically now.

**Comment by rukai** [comment]
Automatic setup works great!

**Comment by geranim0** [comment]
Also for people on other distributions like debian, dolphin installs in /.local/share/dolphin-emu so you'll want to modify each source file of the root directory of the project to your actual dolphin folder.

**Comment by altf4** [comment]
@geranim0 Does the latest master branch work for you, for setup? It should now automatically figure out where your dolphin directory is and move around files and stuff on its own. Commit: afc025555e28aa0c66be0c7c9f2a5aa39a63c92e

**Comment by altf4** [comment]
I'm going to call this fixed as of afc0255. If you have any other issues, reopen this or make a new issue.
